tasks/db_deploy/migrations/migrate_versions_4_to_5/migrate_sql.py

```diff
@@ -0,0 +1,311 @@
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_orphan_report table.
+    """
+    return text(
+        """
+            -- Create reconcile_orphan_report table
+            CREATE TABLE IF NOT EXISTS reconcile_orphan_report  
+            ( 
+              job_id       	int8 NOT NULL
+            , key_path     	text NOT NULL
+            , etag         	text NOT NULL
+            , last_update  	timestamp with time zone NOT NULL
+            , size_in_bytes	int8 NOT NULL
+            , storage_class	text NOT NULL
+            , CONSTRAINT PK_reconcile_orphan_report PRIMARY KEY(job_id,key_path)
+            , CONSTRAINT FK_reconcile_job_orphan_report FOREIGN KEY(job_id) REFERENCES reconcile_job(id)
+            );
+
+            -- Comment
+            COMMENT ON TABLE reconcile_orphan_report IS 'Table that identifies objects in the ORCA S3 Glacier bucket that are not in the ORCA catalog from the internal reconciliation job.';
+            COMMENT ON COLUMN reconcile_orphan_report.job_id IS 'Associates the orphaned file to a internal reconciliation job. References the reconcile jobs table.';
+            COMMENT ON COLUMN reconcile_orphan_report.key_path IS 'Key that contains the path and file name. Value is obtained from the reconcile_s3_object (key_path) column.';
+            COMMENT ON COLUMN reconcile_orphan_report.etag IS 'AWS Etag of the object. Value is obtained from the reconcile_s3_object (etag) column.';
+            COMMENT ON COLUMN reconcile_orphan_report.last_update IS 'AWS last update of the object. Value is obtained from the reconcile_s3_object (lst_update) column.';
+            COMMENT ON COLUMN reconcile_orphan_report.size_in_bytes IS 'AWS size of the object in bytes. Value is obtained from the reconcile_s3_object (size) column.';
+            COMMENT ON COLUMN reconcile_orphan_report.storage_class IS 'AWS storage class the object is in. Value is obtained from the reconcile_s3_object (storage_class) column.';
+        """
+    )
+
+
+def reconcile_phantom_report_table_sql() -> TextClause:
+    """
+    Full SQL for creating the reconcile_phantom_report table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_phantom_report table.
+    """
+    return text(
+        """
+            -- Create reconcile_phantom_report table
+            CREATE TABLE IF NOT EXISTS reconcile_phantom_report  
+            ( 
+              job_id          	int8 NOT NULL
+            , collection_id   	text NOT NULL
+            , granule_id      	text NOT NULL
+            , filename        	text NOT NULL
+            , key_path        	text NOT NULL
+            , orca_etag       	text NOT NULL
+            , orca_last_update	timestamp with time zone NOT NULL
+            , orca_size       	int8 NOT NULL
+            , CONSTRAINT PK_reconcile_catalog_mismatch_report PRIMARY KEY(job_id,collection_id,granule_id,key_path)
```

Comment:
Bad name from when I made the script. Change to PK_reconcile_phantom_report

Reply:
changed